sdk/mixedreality/Microsoft.Azure.Management.MixedReality/src/Generated/Models/Operation.cs

```diff
@@ -31,19 +31,15 @@ public Operation()
         /// </summary>
         /// <param name="name">Operation name:
         /// {provider}/{resource}/{operation}</param>
+        /// <param name="isDataAction">Indicates whether the operation is a
+        /// data action</param>
         /// <param name="display">The object that represents the
         /// operation.</param>
-        /// <param name="isDataAction">Whether or not this is a data plane
-        /// operation</param>
-        /// <param name="origin">The origin</param>
-        /// <param name="properties">Properties of the operation</param>
-        public Operation(string name = default(string), OperationDisplay display = default(OperationDisplay), bool? isDataAction = default(bool?), string origin = default(string), OperationProperties properties = default(OperationProperties))
+        public Operation(string name = default(string), bool? isDataAction = default(bool?), OperationDisplay display = default(OperationDisplay))
```

Comment:
This is a breaking change, please swap the order of the parameters in the [swagger file](https://github.com/Azure/azure-rest-api-specs/blob/2083db78a2a5a1145fbbd0f25c9a704969c16ebc/specification/mixedreality/resource-manager/Microsoft.MixedReality/preview/2021-03-01-preview/proxy.json#L167).

Also, is removing the `origin` and `properties` an approved breaking change? Can you provide the swagger PR ID as well please?

Reply:
@JonathanCrd Fixed in [this PR](https://github.com/Azure/azure-rest-api-specs/pull/13077) pls review

Reply:
@JonathanCrd Swagger was fixed and code updates.